src/rules/__tests__/normalizeSyntheticEvent.test.js

```diff
@@ -111,4 +111,70 @@ describe('normalizeSyntheticEvent', function () {
       }
     });
   });
+
+  it(
+    'does not override meta of first normalized event if synthetic event is ' +
+      'normalized multiple times',
+    function () {
+      var syntheticEvent = {};
+      var syntheticEventMeta1 = { $rule: { id: 'ABC123' } };
+      var syntheticEventMeta2 = { $rule: { id: 'ABC456' } };
+      var normalizedSyntheticEvent1 = normalizeSyntheticEvent(
+        syntheticEventMeta1,
+        syntheticEvent
+      );
+      var normalizedSyntheticEvent2 = normalizeSyntheticEvent(
+        syntheticEventMeta2,
+        syntheticEvent
+      );
+
+      expect(normalizedSyntheticEvent1.$rule.id).toEqual('ABC123');
+      expect(normalizedSyntheticEvent2.$rule.id).toEqual('ABC456');
```

Comment:
It's probably worthwhile to also assert that `syntheticEvent` is still an empty object.